app/src/db/migrations/20200514115312_001-forms.js

```diff
@@ -0,0 +1,20 @@
+exports.up = function(knex) {
+  return Promise.resolve()
+    .then(() => knex.schema.createTable('form', table => {
+      table.uuid('formId').primary();
+      table.string('name').unique().notNullable();
+      table.string('slug').unique().notNullable().comment('This will be used for paths and user interactions');
+      table.boolean('public').notNullable().defaultTo(true);
+      table.specificType('keywords', 'text ARRAY');
+      table.string('prefix').unique().notNullable().comment('This must indicate the business prefix');
```

Comment:
In comment for prefix maybe note it's used for DB table prefix.

Reply:
done